Gemfile.lock

```diff
@@ -313,7 +314,7 @@ GEM
     ice_nine (0.11.2)
     iniparse (1.4.2)
     jmespath (1.3.1)
-    json (2.1.0)
```

Comment:
Note that I came in here and manually changed the `Gemfile.lock` since the Mandrill API gem has `json >= 1.7.7 < 2.0` as a dependency.

[There is a PR](https://bitbucket.org/mailchimp/mandrill-api-ruby/pull-requests/8/fix-json-version/diff) to fix to bump the json version number, but it looks like it has been open since December 2016 with lots of +1s and little movement.